scapy/config.py

```diff
@@ -378,8 +378,8 @@ def _version_checker(module, minver):
 
 def isCryptographyValid():
     """
-    Check if the cryptography library is present, and if it is recent enough
-    for most usages in scapy (v1.7 or later).
+    Check if the cryptography module >= 1.7 is present. This is the minimum
```

Comment:
Actually we've dropped <2.0.0
https://github.com/secdev/scapy/commit/b6589098cca0fda17efa9e1b0181365b53128196
The PR was using some >2.0.0 only things.

Reply:
What should I do? Remove this function?

Reply:
The function still checks if cryptography is present. We could issue a warning if the version is < 2.0.0 (and update it to return True only when > 2.0.0)

Reply:
I updated the PR.